### ISSUE TITLE
Load from Google Drive blocked by Google

### ISSUE THREAD

**Issue by avagp** [photopea/photopea#4114]
I cannot open it.

it  says that the app was blocked by google

![image](https://user-images.githubusercontent.com/11801254/145891276-efe6e391-bf02-41a4-a8b8-e37c0b543155.png)

**Comment by photopea** [comment]
In my case, there is a button "Advanced", which you can click, and allow the access.

**Comment by avagp** [comment]
No. I know what are you referring to. 

But it is not there.

**Comment by tomayac** [comment]
The app is currently [not verified](https://support.google.com/cloud/answer/7454865?hl=en). You need to go through the [verification steps](https://support.google.com/cloud/answer/7454865?hl=en#:~:text=through%20verification%20again.-,Verification%C2%A0for%20apps,-Before%20you%20start) and get the app verified.

**Comment by photopea** [comment]
@tomayac Yes, I know. But I have been seeing this for months it my Console. 
![image](https://user-images.githubusercontent.com/15955558/146038772-01023f00-e3ec-42f4-b106-5ad5f977735a.png)

I feel desperate, as there is nobody I can get in touch with, to find out what is actually going on, and if it will ever be approved.

**Comment by tomayac** [comment]
I have asked around if anyone can help. Waiting for a response.

**Comment by tomayac** [comment]
@photopea: Can you send me the project number? tomac @ google . com. Thanks!

**Comment by photopea** [comment]
There is an ID: "photopea-1265". Where can I see the project number?

**Comment by tomayac** [comment]
Lets see if this helps the support team.

**Comment by tomayac** [comment]
The team have sent you an email. Please follow up as advised in there.

**Comment by photopea** [comment]
Yes, we are exchanging emails, thanks a lot! :)

**Comment by photopea** [comment]
@tomayac we have exchanged many emails with them. The current problem is, that something is wrong or not clear with my Privacy Policy. I am trying to express, that I am not storing any info about users at all. There is a piece of JS running in a browser of a person, which communicates with a GDrive of that same person. No data ever goes to any server of Photopea.

I wish it was enough for Google, that the person says "I trust Photopea", to let them be connected to their Google Drive. Now, it almost seems like there are only some rich VIP companies, which can afford expensive lawyers and developers, who would read through 4896 pages of Google's verification procedures, and prepare the correct privacy policy and make the UI look trustworthy in some way. I am not sure if I have a chance to achieve it :(

**Comment by tomayac** [comment]
I remember that it was a hassle for me, too, but here's a privacy policy that worked for one of my extensions: https://github.com/tomayac/wikipedia-tools-for-google-spreadsheets/blob/master/PRIVACY.md. I suppose something simple like this can work for your case, too.

**Comment by photopea** [comment]
So now, I have received the identical email from you about five times, telling me, that my privacy policy should be accessible from the main site. But I reply, that it is, but then, I again receive the identical email from you.

Do you know if it is a person or a robot replying at the other side? I wish they could at least write a few words about what exactly is the problem, instead of sending the same emails five times.

BTW. Photopea has been confirmed by the Dropbox and now, any user can fully access their dropbox in Photopea.

**Comment by tomayac** [comment]
I have asked the team to clarify what the actual problem is.

**Comment by photopea** [comment]
@avagp  Could you try it again? Is it still blocked?

**Comment by Alpasc** [comment]
> @avagp Could you try it again? Is it still blocked?

I just tried and it is still blocked

**Comment by avagp** [comment]
yes, still bloacked

**Comment by photopea** [comment]
Is it working now?

**Comment by avagp** [comment]
YES IT DOES!!
[image: image.png]

**Comment by avagp** [comment]
:)

**Comment by tomayac** [comment]
Happy this issue could finally be resolved.
